AnimalsTest/AnimalsTest.csproj

```diff
@@ -35,17 +35,11 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <!--<ItemGroup>
-    <EditorConfigFiles Remove="D:\Natalia\Project1\WIT\AnimalsTest\AnimalsTest\.editorconfig" />
-  </ItemGroup>-->
-
   <ItemGroup>
-    <EditorConfigFiles Include=".editorconfig"/>     
+    <EditorConfigFiles Include=".editorconfig"/>
```

Comment:
Mentioning editor config in csproj is not necessary. 
The editorconfig placed in the root of solution will be used: https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022#file-hierarchy-and-precedence

VS might add this item if editor config is not in hierarchy.